.env.example

```diff
@@ -3,6 +3,9 @@ MAINNET_PRIVATE_KEY=
 TEST_CLAIM_SIGNER_PRIVATE_KEY=
 MAINNET_CLAIM_SIGNER_PRIVATE_KEY=
 
+ALCHEMY_ARBITRUM_RPC=
+ALCHEMY_SEPOLIA_RPC=
```

Comment:
Gotta add these to the github settings so the test pass - I can do this if needed just lmk